autoload/preview_markdown.vim

```diff
@@ -31,28 +31,48 @@ function! preview_markdown#preview() abort
     return
   endif
 
-  if !has('terminal')
+  if !has('terminal') && !has('nvim')
     call s:echo_err('this version doesn''t support terminal')
     return
   endif
 
-  let opt = {
-        \ 'in_io': 'file',
-        \ 'in_name': tmp,
-        \ 'exit_cb': function('s:remove_tmp', [tmp]),
-        \ 'vertical': get(g:, 'preview_markdown_vertical', 0),
-        \ }
+  if has('nvim')
+    if get(g:, 'preview_markdown_vertical', 0) is 1
+      vnew
+    else
+      new
+    endif
```

Comment:
`g:preview_markdown_vertical` is not defined by default, so default behavior is horizontal opening.
But, sample image in README.md is vertical openning.
I would suggest `g:preview_markdown_vertical` is 1 by default, or it horizontally open if `g:preview_markdown_horizontal` is 1.
This settings will be intuition for users reading README.md.

Reply:
I don't want to add too many options,
so I think better that vertical when preview_markdown_vertical is 1 and horizontal when undefined.